### PR TITLE
3.106

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

**Comment by VincentLanglet** [comment]
@jordisala1991 This can be released and merge into 4.x before 4.2 too

**Review by jordisala1991 — DISMISSED** [review]

**Review by core23 — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.106.0](https://github.com/sonata-project/SonataAdminBundle/compare/3.105.3...3.106.0) - 2021-10-30
+### Added
+- [[#7563](https://github.com/sonata-project/SonataAdminBundle/pull/7563)] AbstractAdmin::removeExtension() ([@VincentLanglet](https://github.com/VincentLanglet))
+- [[#7564](https://github.com/sonata-project/SonataAdminBundle/pull/7564)] The ability to not add a `select` column when accessing to the List with AJAX. ([@VincentLanglet](https://github.com/VincentLanglet))
```
Comment (phansys) [inline CHANGELOG.md]:
```suggestion
- [[#7564](https://github.com/sonata-project/SonataAdminBundle/pull/7564)] The ability to not add a `select` column when accessing to the list with AJAX ([@VincentLanglet](https://github.com/VincentLanglet))
```

**Review by phansys — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,19 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.106.0](https://github.com/sonata-project/SonataAdminBundle/compare/3.105.3...3.106.0) - 2021-10-30
+### Added
+- [[#7563](https://github.com/sonata-project/SonataAdminBundle/pull/7563)] AbstractAdmin::removeExtension() ([@VincentLanglet](https://github.com/VincentLanglet))
```
Comment (phansys) [inline CHANGELOG.md]:
```suggestion
- [[#7563](https://github.com/sonata-project/SonataAdminBundle/pull/7563)] `AbstractAdmin::removeExtension()` ([@VincentLanglet](https://github.com/VincentLanglet))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,19 @@
...
+- [[#7564](https://github.com/sonata-project/SonataAdminBundle/pull/7564)] The ability to not add a `select` column when accessing to the List with AJAX. ([@VincentLanglet](https://github.com/VincentLanglet))
+- [[#7567](https://github.com/sonata-project/SonataAdminBundle/pull/7567)] Added final information to `AbstractAdmin::getAccess()` ([@core23](https://github.com/core23))
```
Comment (phansys) [inline CHANGELOG.md]:
```suggestion
- [[#7567](https://github.com/sonata-project/SonataAdminBundle/pull/7567)] Final information to `AbstractAdmin::getAccess()` ([@core23](https://github.com/core23))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,19 @@
...
+
+### Fixed
+- [[#7529](https://github.com/sonata-project/SonataAdminBundle/pull/7529)] Fixed setting a default order if it is not set in the field description ([@franmomu](https://github.com/franmomu))
```
Comment (phansys) [inline CHANGELOG.md]:
```suggestion
- [[#7529](https://github.com/sonata-project/SonataAdminBundle/pull/7529)] Setting a default order if it is not set in the field description ([@franmomu](https://github.com/franmomu))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,19 @@
...
+- [[#7530](https://github.com/sonata-project/SonataAdminBundle/pull/7530)] Fixed using multiple properties with `ModelAutocompleteType` with different order ([@franmomu](https://github.com/franmomu))
```
Comment (phansys) [inline CHANGELOG.md]:
```suggestion
- [[#7530](https://github.com/sonata-project/SonataAdminBundle/pull/7530)] Using multiple properties with `ModelAutocompleteType` with different order ([@franmomu](https://github.com/franmomu))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,19 @@
...
+
+### Removed
+- [[#7567](https://github.com/sonata-project/SonataAdminBundle/pull/7567)] Removed final information from `AbstractAdmin::getAccessMapping()` ([@core23](https://github.com/core23))
```
Comment (phansys) [inline CHANGELOG.md]:
```suggestion
- [[#7567](https://github.com/sonata-project/SonataAdminBundle/pull/7567)] Final information from `AbstractAdmin::getAccessMapping()` ([@core23](https://github.com/core23))
```

**Review by phansys — DISMISSED** [review]

**Review by phansys — APPROVED** [review]

**Pull request merged**